Common/src/Common/Service/Table/Formatter/ExternalConversationLinkFactory.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Common\Service\Table\Formatter;
+
+use Interop\Container\ContainerInterface;
+use Laminas\ServiceManager\Factory\FactoryInterface;
+
+class ExternalConversationLinkFactory implements FactoryInterface
+{
+    /**
+     * @param  ContainerInterface $container
+     * @param  $requestedName
+     * @param  array|null         $options
+     * @return InternalLicenceConversationLink
+     */
+    public function __invoke(ContainerInterface $container, $requestedName, array $options = null)
+    {
+        $urlHelper = $container->get('Helper\Url');
```

Comment:
UrlHelperService::class here would be preferable. 

```suggestion
        $urlHelper = $container->get(UrlHelperService::class);
```